public/locales/en/register.json

```diff
@@ -45,12 +45,25 @@
     "info": {
       "heading": "Before we start",
       "subheading": "Registering your name takes three steps",
-      "items": [
+      "ethItems": [
         "Complete a transaction to begin the timer",
         "Wait 60 seconds for the timer to complete",
         "Complete a second transaction to secure your name"
       ],
-      "setupProfile": "I'd like to set up my profile first"
+      "moonpayItems": [
+        "Create or login to an existing moonpay account",
+        "Use your credit or debit card to complete a single transaction"
+      ],
+      "setupProfile": "I'd like to set up my profile first",
+      "paymentMethod": "Payment method",
+      "notEnoughEth": "Not enough ETH in wallet",
+      "creditOrDebit": "Credit or debit card",
+      "additionalFee": "Additional processing fee",
+      "pendingMoonpayTransaction": "Your moonpay transaction is processing. This may take up to two minutes. You can check your progress from the confirmation email you recieved.",
```

Comment:
```suggestion
      "pendingMoonpayTransaction": "Your moonpay transaction is processing. This may take up to two minutes. You can check your progress from the confirmation email you received.",
```